source/fundamentals/data-formats/pojo-customization.txt

```diff
@@ -299,9 +299,12 @@ package:
        and/or deserialize a property.
 
    * - ``BsonProperty``
-     - Specifies a custom document field name when converting the POJO
-       field to BSON. You can include a discriminator to serialize POJOs
-       nested within the field.
+     - | Specifies a custom document field name when converting the POJO
+         field to BSON. You can include a discriminator to serialize POJOs
+         nested within the field.
+       | When applying ``@BsonProperty`` to a private field, you must also add
+         getter and setter methods for that field to serialize and customize
+         the field name.
```

Comment:
I notice some of the other cells in this table have a Tip box enclosed – might be a good idea to enclose the second paragraph in a Note box?

Reply:
I like that idea - updated!